README.md

```diff
@@ -1,83 +1,87 @@
-# Thunderbird Sieve Extension
+# [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) for [Thunderbird](https://www.thunderbird.net/)
 
-[Sieve](http://en.wikipedia.org/wiki/Sieve_%28mail_filtering_language%29) is
-a powerful script language for server-side mail filtering. It is
-intended to be used with [IMAP](http://tools.ietf.org/html/rfc3501) and thus
-it is widely spread. Many IMAP Server are capable of running sieve filters.
-Sieve stores and runs all script on the server-side.
+[Sieve](http://en.wikipedia.org/wiki/Sieve_%28mail_filtering_language%29) is a powerful scripting language for server-side mail filtering. It is intended to be used with [IMAP](http://tools.ietf.org/html/rfc3501) which is ubiquitous. Many IMAP Servers are capable of running Sieve filters. Sieve stores and runs all scripts on the server-side.

```

Comment:
Lines have to break at 40 or 80 characters in length so that 
you can read it with a plain text editor or on a large screen.

Reply:
Will change as you requested. However, there is a "Soft wrap" option in GitHub which enable view of the code on any type of monitor. If you would like to change back please advise.